infra/kops/TEMPLATE_CLUSTER/cluster.yml

```diff
@@ -3,36 +3,60 @@ kind: Cluster
 metadata:
   name: {{ .cluster_name }}
 spec:
+  additionalPolicies:
+    node: |
+      [
+        {
+          "Action": [
+            "sts:AssumeRole"
+          ],
+          "Effect": "Allow",
+          "Resource": "*"
+        },
+        {
+          "Effect": "Allow",
+          "Action": [
+            "ecr:DescribeImages",
+            "ecr:InitiateLayerUpload",
+            "ecr:UploadLayerPart",
+            "ecr:CompleteLayerUpload",
+            "ecr:PutImage",
+            "ecr:CreateRepository"
+          ],
+          "Resource": "*"
+        }
+      ]
   api:
     loadBalancer:
       type: Public
+  authorization:
+    alwaysAllow: {}
   channel: alpha
   cloudProvider: aws
   configBase: {{ .kops_state_store }}/{{ .cluster_name }}
   dnsZone: {{ .dns_zone_id }}
   etcdClusters:
   - etcdMembers:
-    - instanceGroup: master-eu-west-1a
-      name: a
-    - instanceGroup: master-eu-west-1b
-      name: b
-    - instanceGroup: master-eu-west-1c
-      name: c
+{{- range split .availability_zones "," }}
+    - instanceGroup: master-{{ . }}
+      name: {{ . }}
```

Comment:
Does it matter we're changing the names here? (e.g. does something else relies on these names?)

Reply:
Actually the contents of `TEMPLATE_CLUSTER` aren't used at all anymore - I should remove this.

Reply:
Nice